server/runner/lib/runner/qa/agent.ex

```diff
@@ -493,7 +493,7 @@ defmodule Runner.QA.Agent do
       {:ok, devices} ->
         device =
           Enum.find(devices, fn device ->
-            device.name == "iPhone 16" and
+            device.name == "iPhone 17" and
```

Comment:
my thoughts on this were ... why not 😁 

obviously, eventually the device should be made configurable down the line